packages/e2e-tests/tests/balancer/bal.swap.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from '@playwright/test'
+
+test('Open swap page and try eth wrap', async ({ page }) => {
+  await page.goto('http://localhost:3000/')
+  await page.getByRole('link', { name: 'Swap', exact: true }).click()
+
+  // Selects Wrapped Ether token out
+  await page.getByRole('button', { name: 'Select token' }).click()
+  await page.getByPlaceholder('Search by name, symbol or').click()
```

Comment:
focus should be set here on modal open or doesn't that work for playwright?

Reply:
Probably yes. We can improve this things when we have more bandwidth.